app/assets/stylesheets/main.scss

```diff
@@ -1,7 +1,7 @@
 @import 'colors';
 @import 'application.erb';
 @import 'movie-card';
-
+@import 'profile-page';
```

Comment:
this main.scss file shouldn't need to import all your style sheets - that is the responsibility of the `application.css` file. the only time you need to import your css/scss is when you want to say import the colors file into another stylesheet to use the color variables